CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 ## Unreleased
 - Update to symplify/easy-coding-standard ^10.0
+- Allow using `slevomat/coding-standard` ^7
+- Use forked version of ecs and php-cs-fixer with a better support for PHP 8+
```

Comment:
@MortalFlesh this is probably copied by mistake from https://github.com/lmc-eu/php-coding-standard/pull/75/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed ?